charts/rucio-webui/README.md

```diff
@@ -97,15 +97,114 @@ data:
+kubectl create secret generic <releasename>-server-hostcert --from-file=hostcert.pem=/path/to/hostcert.pem
 kubectl create secret generic <releasename>-server-hostkey --from-file=hostkey.pem=/path/to/hostkey.pem
-kubectl create secret generic <releasename>-server-cafile --from-file=ca.pem=/path/to/ca.pem
 ```
+
+### Mount Secrets
+
+You can mount the secrets into the WebUI pod using the `secretMounts` section. For example:
+
+```yaml
+secretMounts:
+  - name: <releasename>-server-hostcert
+    mountPath: /etc/grid-security/hostcert.pem
+    subPath: hostcert.pem
+  - name: <releasename>-server
+    mountPath: /etc/grid-security/hostkey.pem
+    subPath: hostkey.pem
+```
+
+### CA bundles
+
+There are two types of CA bundles that are used by the WebUI:
+1. For verification of the x509 client certificates, which is done by Apache by setting the `SSLCACertificateFile` or `SSLCACertificatePath` directive in apache configuration.
+2. For verification of the host certificates of the Rucio server, which is done by NodeJS, ultimately by setting the `RUCIO_WEBUI_SERVER_CA_BUNDLE` environment variable in the WebUI container.
+
+
+#### CA bundles for x509 client certificates
+##### Multiple CA files
+For x509 based authentication, if you have multiple pem encoded CA files, you must mount them at the path specified by the `RUCIO_CA_PATH` environment variable.
+
+You must also set the `RUCIO_CA_PATH` environment variable using the `addtionalEnvs` section to point to the directory where the CA files are mounted. For example, to mount the CA files at `/etc/grid-security/certificates/` and set the `RUCIO_CA_PATH` environment variable to `/etc/grid-security/certificates`, you would set the following in the `values.yaml` file:
+
+```yaml
+hostPathMounts:
+  - hostPath: /etc/grid-security/certificates/
+    mountPath: /etc/grid-security/certificates/
+    readOnly: true
+    type: DirectoryOrCreate
+
+additionalEnvs:
+  - name: RUCIO_CA_PATH
+    value: "/etc/grid-security/certificates"
+```
+
+The hostPathMounts is just one way to provide the CA files when you have the CA files locally available on your kubernetes nodes. You can also use other methods like `configmaps` or `secrets` to provide the CA files. Please make sure that the CA files are mounted at the path specified by the `RUCIO_CA_PATH` environment variable.
+
+##### Single CA file
+
+If you wish to use a single CA file for x509 authentication, you can provide the CA file as a secret in the same namespace as the WebUI pod, you must mount the file at the path specified by the `RUCIO_CA_FILE` environment variable.
+
+You must also set the `RUCIO_CA_FILE` environment variable using the `addtionalEnvs` section to point to the directory where the CA file is mounted. For example, to mount the CA file at `/etc/grid-security/certificates/ca.pem` and set the `RUCIO_CA_FILE` environment variable to `/etc/grid-security/certificates/ca.pem`, you would set the following in the `values.yaml` file:
```

Comment:
```suggestion
You must also set the `RUCIO_CA_FILE` environment variable using the `additionalEnvs` section to point to the directory where the CA file is mounted. For example, to mount the CA file at `/etc/grid-security/certificates/ca.pem` and set the `RUCIO_CA_FILE` environment variable to `/etc/grid-security/certificates/ca.pem`, you would set the following in the `values.yaml` file:
```

Reply:
Haha, thanks! I need new batteries in my keyboard, it skips too many letters